### ISSUE TITLE
Need a way to refresh page if webview is unresponsive

### ISSUE THREAD

**Issue by cipherzzz** [Musicoin/desktop#178]
Seeing this - we should catch this and show some type of dialog to allow the user the change to try their request again.

![image](https://user-images.githubusercontent.com/33289322/36451131-ff5b9926-165d-11e8-886d-9e583b2a56e3.png)

**Comment by Varunram** [comment]
Agreed, we need some mechanism for users to be able to refresh the page in case something like this happens.

**Comment by etaletai13** [comment]
It took me so long to realize you meant automatically ...
